### PR TITLE
Edit Fullnode Docker README

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: docker/fullnode/README.md [hunk 1]
```diff
@@ -1,83 +1,93 @@
...
 
-Get the latest version of the fullnode config [here](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml), or:
+Download the latest version of the fullnode config [fullnode-template.yaml](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml) over the web or by using `curl` or `wget`, for example:
 
-```wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml```
+```shell
+wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml
+```
+
+TODO: Don't they need to rename this file to `fullnode.yaml` too?
```
Comment (tharbert) [inline docker/fullnode/README.md:25]: Technically no as I configured it to require the least steps possible to get up and running, see: 

https://github.com/MystenLabs/sui/blob/e03a0af538d6d70a9744012438192950ea7d5599/docker/fullnode/docker-compose.yaml#L13

File: docker/fullnode/README.md [hunk 2]
```diff
@@ -1,83 +1,93 @@
...
-To use the DevNet explorer with your fullnode follow these steps:
+To use the Sui Explorer with your fullnode, follow these steps:
 - Open https://explorer.devnet.sui.io
-- Click the green Devnet button in the top right
+- Click the green **Devnet** button in the top right
 - Select 'Custom RPC URL'
 - Set it to http://127.0.0.1:9000
 
-# troubleshoot / tips / documentation
+TODO: Confirm this shouldn't be Local as in the existing fullnode doc.
```
Comment (tharbert) [inline docker/fullnode/README.md:70]: At least for me when I attempt to use 'Local' with my fullnode instance it doesn't work. I need to explicitly point the explorer at the 'Custom RPC URL' http://127.0.0.1:9000

I do remember seeing some chatter about the Local default being changed however I am not sure if this has or is happening.. /cc @666lcz / @stella3d ?

File: docker/fullnode/README.md (hunk 1 above)
Comment (tharbert) [reply docker/fullnode/README.md:25]: If we want consistency we can add a step to rename/move the template prior to starting docker-compose however if its not necessary then i prefer one less step.

File: docker/fullnode/README.md [hunk 3]
```diff
@@ -1,83 +1,93 @@
...
 
 Take everything down, removing the container and volume. Use this to start completely fresh (image, config, or genesis updates):
 
 ```docker-compose down --volumes```
 
-## inspect the state of a running fullnode
+### Inspect the state of a running fullnode
 
-Get the running container id:
+Get the running container ID:
```
Comment (allan-bailey) [inline docker/fullnode/README.md:90]: nit:  'docker ps' shows all the containers running.

File: docker/fullnode/README.md [hunk 4]
```diff
@@ -89,9 +99,9 @@ Inspect the database:
 
 ```ls -la suidb/```
 
-## local rpc connectivity issues
+### Investigate local RPC connectivity issues
 
-Update the json-rpc-address in the fullnode config to listen on all addresses:
+Update the `json-rpc-address` in the fullnode config to listen on all addresses:
```
Comment (allan-bailey) [inline docker/fullnode/README.md:104]: When running as a Docker container the container must listen on 0.0.0.0:<port> in order for the 'export' of that port to work to the host.   localhost isn't exported.

File: docker/fullnode/README.md [hunk 5]
```diff
@@ -100,17 +110,17 @@ Update the json-rpc-address in the fullnode config to listen on all addresses:
 +json-rpc-address: "0.0.0.0:9000"
 ```
 
-## install wget and curl
+### Install wget and curl
 
-On MacOS using [homebrew](https://brew.sh/):
+Download each package. For examp,e, on macOS use [homebrew](https://brew.sh/):
```
Comment (allan-bailey) [inline docker/fullnode/README.md:115]: tyop

File: docker/fullnode/README.md (hunk 1 above)
Comment (Clay-Mysten) [reply docker/fullnode/README.md:25]: Perfect. Thank you!

Comment (Clay-Mysten) [reply docker/fullnode/README.md]: Removed TODO.

File: docker/fullnode/README.md (hunk 5 above)
Comment (Clay-Mysten) [reply docker/fullnode/README.md]: Fixed. Thanks for catching.

File: docker/fullnode/README.md (hunk 3 above)
Comment (Clay-Mysten) [reply docker/fullnode/README.md:88]: Should this be added to the doc?

File: docker/fullnode/README.md (hunk 4 above)
Comment (Clay-Mysten) [reply docker/fullnode/README.md:102]: Is an update needed here then?

File: docker/fullnode/README.md (hunk 3 above)
Comment (allan-bailey) [reply docker/fullnode/README.md:88]: no. just a nit.

File: docker/fullnode/README.md [hunk 6]
```diff
@@ -1,83 +1,91 @@
...
-# use your fullnode with explorer 
+## Use your fullnode with Explorer 
 
-To use the DevNet explorer with your fullnode follow these steps:
+To use the Sui Explorer with your fullnode, follow these steps:
 - Open https://explorer.devnet.sui.io
-- Click the green Devnet button in the top right
+- Click the green **Devnet** button in the top right
 - Select 'Custom RPC URL'
 - Set it to http://127.0.0.1:9000
```
Comment (666lcz) [inline docker/fullnode/README.md:66]:
```suggestion
- Select 'Local'
```

**Pull request merged**